### ISSUE TITLE
Policies marked as "Show In Dashboard" are not appearing in the dashboard

### ISSUE THREAD

**Issue by iamdamosuzuki** [bavc/signalserver#284]
Am I correct to assume that if a policy is marked as "Show in Dashboard" it should appear as a bullet under "Policies" on the dashboard? That is not happening currently

**Comment by yayoiukai** [comment]
yes, we haven't implemented yet. We haven't decided that what we wanted to put on the dashboard. But I think we can now.
